ds-prep-python-conditionals/main.py

```diff
@@ -0,0 +1,55 @@
+day_of_week = "saturday"
+
+if day_of_week == "saturday" or "sunday":
+  print("Have a good weekend")
+else:
+  print("It's a weekday!")
+
+student_1_score = 5
+
+if student_1_score >= 70:
+  pass_or_fail = "You passed"
+else:
+  pass_or_fail = "You Failed!"
+
+print(student_1_score, pass_or_fail)
+
+student_2_score = 76
+
+if student_2_score < 60:
+  letter_grade = "F"
+elif student_2_score >= 60 and student_2_score < 70:
+  letter_grade = "D"
+elif student_2_score >= 70 and student_2_score < 80:
+  letter_grade = "C"
+elif student_2_score >= 80 and student_2_score < 90:
+  letter_grade = "B"
+elif student_2_score >= 90 and student_2_score < 100:
+  letter_grade = "A"
+else:
+  student_2_score >= 100
+  letter_grade = "A+"
```

Comment:
The `student_2_score` comparison isn't necessary in the `else` section.

Reply:
![image](https://user-images.githubusercontent.com/54218913/149637387-240173f8-8584-41a1-8a21-617af0bd7345.png)